ferrocene/ci/docker-images/ubuntu-20/Dockerfile

```diff
@@ -102,15 +102,15 @@ RUN mkdir /opt/nodejs \
 ENV PATH "/opt/nodejs/bin:$PATH"
 
 # Install AWS CLI from the published binaries, as no source tarball is provided
-# and the version in the Ubuntu 18.04 archive is too old and doesn't support
+# and the version in the Ubuntu 20.04 archive is too old and doesn't support
 # `aws ecr get-login-password`.
 RUN curl -Lo /tmp/awscli.zip https://awscli.amazonaws.com/awscli-exe-linux-x86_64-2.12.4.zip \
```

Comment:
`get-login-password` is in a version available from 20.04... maybe we can try that in a future pr

Reply:
Ubuntu 20.04 still has AWS CLI 1.x, we need 2.x.